Modules/Core/Common/include/itkVariableLengthVector.hxx

```diff
@@ -28,9 +29,8 @@ namespace itk
 
 template <typename TValue>
 VariableLengthVector<TValue>::VariableLengthVector(unsigned int length)
-  : m_Data(nullptr)
+  : m_Data(new TValue[length])
 {
-  Reserve(length);
```

Comment:
Simplified the `VariableLengthVector(length)` constructor by removing its internal `Reserve(length)` call.